shap/explainers/_kernel.py

```diff
@@ -150,7 +150,7 @@ def _convert_symbolic_tensor(symbolic_tensor) -> np.ndarray:
                 tensor_as_np_array = sess.run(symbolic_tensor)
         return tensor_as_np_array
 
-    def __call__(self, X):
+    def __call__(self, X, l1_reg="auto"):
```

Comment:
Would it be possible to adjust a test using the call interface to actually use this argument? That would be great IMO

Reply:
Good idea, done. Would you kindly re-review?